README.md

```diff
@@ -31,17 +31,12 @@ glow help <command> # View help for a command
 
 The full, in-depth tutorial may be found [here](https://glow-lang.org/docs/Glow_Tutorial.html).
 
-### Being worked on
-
-You can watch our [GitHub repository](https://github.com/Glow-Lang/glow)
-for current developments.
```

Comment:
As @plotnick mentioned in #411, the link to the GitHub repo does still make sense, since the README can be viewed from a local checkout, not just on GitHub.